test/openAPI/features/wallet_consent.feature

```diff
@@ -0,0 +1,33 @@
+@method=POST @endpoint=/linked-authorization/consent
+Feature: The endpoint to send the accepted user claims and permitted scopes.
+
+  @smoke @unit @positive
+  Scenario: Successfully sends the accepted user claims and permitted scopes smoke type test
+    Given Wants to send the accepted user claims and permitted scopes
+    And The linkedTransactionId is generated before POST /linked-authorization/consent
+    When Send POST /linked-authorization/consent request with given requestTime and linkedTransactionId
+    Then Receive a response from the /linked-authorization/consent endpoint
+    And The /linked-authorization/consent endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/consent endpoint response should have status 200
+
+  @positive
+  Scenario: Successfully sends the accepted user claims and permitted scopes
+    Given Wants to send the accepted user claims and permitted scopes
+    When Send POST /linked-authorization/consent request with given requestTime, linkedTransactionId, "openid" as permittedAuthorizeScopes, "given_name" as acceptedClaims
```

Comment:
permittedAuthorizeScopes should be from the supported list of scopes from the implementation, can we take this from configuration ?

Reply:
Ok, can you help me understand which configuration you mean?

Reply:
Is there a feature to input a configuration file to test harness based on BB implementation which changes the test slightly for optional elements ?

Reply:
No, we do no have any

Reply:
Since we discussed on multiple PRs, expecting this configurability to come in later, marking this as resolved.